crates/lib/src/action/outcome_worker.rs

```diff
@@ -51,10 +52,37 @@ impl OutcomeWorker {
 		gencheck: Arc<AtomicUsize>,
 		errors_c: mpsc::Sender<RuntimeError>,
 		events_c: priority::Sender<Event, Priority>,
+	) {
+		let commands = working.borrow().commands.clone();
+		Self::spawn_with_commands(
```

Comment:
Hmm okay this is getting ridiculous so I'll just make a breaking release. Here, let's modify the original spawn.

Reply:
haha, okie! how many things are you willing to break with a breaking release? would it make sense to add in more things if a release is already breaking, or would it be better to limit the breaking changes to one or two at a maximum?

Reply:
nah let's go for gold... now that that's open I'm planning to do the command+grouped changes I mentioned earlier after this PR lands! so more is fine

Reply:
haha, going for gold! i love this ! well, then full steam ahead hehe?